src/shogun/kernel/PeriodicKernel.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+ * This software is distributed under BSD Clause 3 license (see LICENSE file).
+ *
+ * Written (W) 2015 Esben Soerig
+ */
+
+#include "PeriodicKernel.h"
```

Comment:
@sorig hey just a minor comment. Can you please follow the same notion when using `#include` here? As in, use `#include <shogun/kernel/PeriodicKernel.h>` instead. Also, it would be better to use the same license in the header. For example, check [FeatureSelection.h](https://github.com/shogun-toolbox/shogun/blob/develop/src/shogun/preprocessor/FeatureSelection.h).